images/linux/scripts/SoftwareReport/SoftwareReport.Generator.ps1

```diff
@@ -110,6 +119,7 @@ $toolsList = @(
     (Get-TerraformVersion),
     (Get-UnZipVersion),
     (Get-WgetVersion),
+    (Get-YamllintVersion),
```

Comment:
What about Ubuntu 16.04?

Reply:
It's worked on 16.04, because Yamllint is installed as apt package.